issues.go

```diff
@@ -33,62 +33,67 @@ type IssuesService struct {
 	timeStats *timeStatsService
 }
 
+// IssueAuthor represents a author of the issue.
+type IssueAuthor struct {
+	ID        int    `json:"id"`
+	State     string `json:"state"`
+	WebURL    string `json:"web_url"`
+	Name      string `json:"name"`
+	AvatarURL string `json:"avatar_url"`
+	Username  string `json:"username"`
+}
+
+// IssueAssignee represents a assignee of the issue.
+type IssueAssignee struct {
+	ID        int    `json:"id"`
+	State     string `json:"state"`
+	WebURL    string `json:"web_url"`
+	Name      string `json:"name"`
+	AvatarURL string `json:"avatar_url"`
+	Username  string `json:"username"`
+}
+
+// IssueAssignees represents collections of assignees of the issue.
+type IssueAssignees []IssueAssignee
+
+// IssueLinks represents links of the issue.
+type IssueLinks struct {
+	Self       string `json:"self"`
+	Notes      string `json:"notes"`
+	AwardEmoji string `json:"award_emoji"`
+	Project    string `json:"project"`
+}
```

Comment:
I'm not sure if there is much value in specifying these embedded struct as specific types, but I guess it doesn't hurt either.

The only one that I would leave out is `IssueAssignees []IssueAssignee`. That one really doesn't add any value as `[]IssueAssignee` already is a specific type.

Reply:
I observer that it helps a lot in writing a unit test so just change struct to specific struct type but ya i can replace ```IssueAssignees []IssueAssignee``` to ```[]*IssueAssignee```